server/graphql/programs/src/types/encounter.rs

```diff
@@ -29,23 +29,23 @@ pub struct EncounterNode {
 #[serde(rename_all = "SCREAMING_SNAKE_CASE")] // only needed to be comparable in tests
 pub enum EncounterNodeStatus {
     Scheduled,
-    Done,
+    Completed,
     Cancelled,
 }
 
 impl EncounterNodeStatus {
     pub fn to_domain(self) -> EncounterStatus {
         match self {
             EncounterNodeStatus::Scheduled => EncounterStatus::Scheduled,
-            EncounterNodeStatus::Done => EncounterStatus::Done,
+            EncounterNodeStatus::Completed => EncounterStatus::Completed,
             EncounterNodeStatus::Cancelled => EncounterStatus::Cancelled,
         }
     }
 
     pub fn from_domain(status: &EncounterStatus) -> EncounterNodeStatus {
         match status {
             EncounterStatus::Scheduled => EncounterNodeStatus::Scheduled,
-            EncounterStatus::Done => EncounterNodeStatus::Done,
+            EncounterStatus::Completed => EncounterNodeStatus::Completed,
```

Comment:
Good -- I changed this from "DONE" to "COMPLETED" in the schema too!

Reply:
thats why I changed it :)